CHANGELOG.md

```diff
@@ -28,6 +29,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - /api/1.1/deliveryservices/hostname/:hostname/sslkeys `GET`
   - /api/1.1/deliveryservices/sslkeys/add `POST`
   - /api/1.1/deliveryservices/xmlId/:xmlid/sslkeys/delete `GET`
+- Instead of rebuilding the riak cluster from scratch every time the riak cluster is only rebuilt when the database is changed.  A bug fix to attach a destructor to the garbage collection of the old cluster fixed a bug where the logs were being spammed by health checking.  Health checking still occurs but it is handled by the cluster itself silently.
```

Comment:
This changelog line should also get moved up to the `Unreleased` section

Reply:
It's in the `[3.0.0]` section, though, and should be in the `[Unreleased]` section above since this is on master. Your other added line above is in the right spot.